front/src/modules/ui/dropdown/components/DropdownMenuSearchInput.tsx

```diff
@@ -38,6 +38,12 @@ export const DropdownMenuSearchInput = forwardRef<
   InputHTMLAttributes<HTMLInputElement>
 >((props, ref) => (
   <StyledDropdownMenuSearchInputContainer>
-    <StyledInput autoComplete="off" placeholder="Search" {...props} ref={ref} />
+    <StyledInput
+      autoFocus
```

Comment:
Hello @AyushAgrawal-A2, thank you for contributing!

So the reason why `autoFocus` is not set to `true` by default in this component is that we could imagine a case where several inputs are needed in one Dropdown.

Could you set the `autoFocus` prop on this component where it is called in the "filter" components, instead of here? (`FilterDropdownEntitySearchInput`, `FilterDropdownNumberSearchInput`, `FilterDropdownTextSearchInput`)

Thank you 🙏

Reply:
@thaisguigon done